raster/r.support/testsuite/test_r_support.py

```diff
@@ -0,0 +1,66 @@
+"""Test of r.support basic functionality
+
+@author Maris Nartiss
+
+@copyright 2021 by Maris Nartiss and the GRASS Development Team
+
+@license This program is free software under the GNU General Public License (>=v2).
+Read the file COPYING that comes with GRASS
+for details
+"""
+import random
+import string
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+from grass.script.core import tempname
+from grass.pygrass.gis import Mapset
+from grass.pygrass import utils
+
+from grass.lib.gis import G_remove_misc
+from grass.lib.raster import Rast_read_bandref
+
+
+class RSupportBandHandlingTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.map = tempname(10)
+        cls.mapset = Mapset().name
+        cls.bandref = "The_Doors"
```

Comment:
😄

Reply:
Wait till you'll see signature file tests :D